oak/server/enclave_server.h

```diff
@@ -166,6 +169,9 @@ class EnclaveServer final : public asylo::TrustedApplication {
   // The gRPC server.
   std::unique_ptr<::grpc::Server> server_ GUARDED_BY(server_mutex_);
 
+  // The port for GRPC set by client; if 0 the server will bind to any available port.
+  int assigned_port_;
```

Comment:
I would expect that `port_` may be `0`, in which case `assigned_port_` will be set to the actual assigned value by the server itself, but I think you are using these with the opposite meaning?

Reply:
You're right, this was sloppy naming on my side :p 

Will rename these